src/bootman/bootman.c

```diff
@@ -446,6 +457,208 @@ bool boot_manager_set_uname(BootManager *self, const char *uname)
         return self->have_sys_kernel;
 }
 
+static bool boot_manager_is_inird_valid_name(const char *name)
```

Comment:
I think the function has a typo for inird -> initrd. Maybe having a little explanation as to why  the name field's first two characters must be numbers (or just general expectations for the filename in a comment.

Reply:
name function: fixed
I added a comment for name explanation